src/storage/s3.rs

```diff
@@ -77,27 +90,52 @@ impl<'a> S3Backend<'a> {
                             content_type: Some(blob.mime.clone()),
```

Comment:
Since you're draining `uploads` anyway, is it now possible not to clone the content?

Reply:
Unfortunately not because of the retrying